src/hooks/useENSRegistrationActionHandler.ts

```diff
@@ -139,25 +136,17 @@ export default function useENSRegistrationActionHandler(
       // we want to speed up the last commit tx sent
       const commitTransactionHash =
         registrationParameters?.commitTransactionHash;
-      const saveCommitTransactionHash = (hash: string) => {
-        dispatch(
-          saveCommitRegistrationParameters({
-            commitTransactionHash: hash,
-          })
-        );
-      };
+
       const tx = getPendingTransactionByHash(commitTransactionHash || '');
       commitTransactionHash &&
         tx &&
         navigate(Routes.SPEED_UP_AND_CANCEL_SHEET, {
           accentColor,
-          onSendTransactionCallback: saveCommitTransactionHash,
```

Comment:
removing since we have this on speed up screen, since we want the speed up to work from the tx list